superset-frontend/src/components/ListView/TableCollection.tsx

```diff
@@ -271,15 +276,16 @@ export default function TableCollection({
                   const columnCellProps = cell.column.cellProps || {};
                   return (
                     <td
-                      className={cx('table-cell', {
+                      data-test="row-cell"
+                      className={cx('row-cell', {
```

Comment:
The `table-cell` class removed here appears to have CSS styles that were applied to it. Is there a reason to cahge it?